js/src/components/contact-information/phone-number-card/verify-phone-number-content.test.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import { act, render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom/extend-expect';
+
+/**
+ * Internal dependencies
+ */
+import VerifyPhoneNumberContent from './verify-phone-number-content';
+
+describe( 'VerifyPhoneNumberContent component', () => {
+	test( 'typing Enter submits enclosing form', async () => {
```

Comment:
:nail_care: As an external consumer of `VerifyPhoneNumberContent` "enclosed form" is an implementation detail, I may not know/care about the `form` inside. Also, the test itself does not test form, but `onVerificationStateChange`

WDYT of rephrasing the description a little?

Reply:
hm... but on the other hand, from UI perspective there is a "form" (even if we don't care about `<form>`)

maybe something like
```suggestion
	test( 'typing Enter submits enclosing form and fires `onVerificationStateChange`', async () => {
```

to at least refer to what we actually precisely test.